alibuild_helpers/git.py

```diff
@@ -1,4 +1,6 @@
+import shlex
```

Comment:
Is this available also in python2.7?

Reply:
No, you're right, I'll change it to use `pipes.quote` on python2, like elsewhere we use the same function.